src/pages/dtrader/_dtrader-tabs.tsx

```diff
@@ -278,7 +278,10 @@ class DtraderTabs extends React.Component {
                             </>
                         </ROW>
                         <EU>
-                            <source src="/Dtrader_GIF_EU.mp4" type="video/mp4" />
+                            <>
+                                <source src="/Dtrader_GIF_EU.mp4" type="video/mp4" />
+                                <source src="/Dtrader_GIF_EU.webm" type="video/webm" />
+                            </>
                         </EU>
```

Comment:
Can this work without the react fragment `<>`?

```
<EU>
  <source src="/Dtrader_GIF_EU.mp4" type="video/mp4" />
  <source src="/Dtrader_GIF_EU.webm" type="video/webm" />
</EU>
```

Reply:
If I remove the fragment I get this error: This JSX tag's 'children' prop expects a single child of type 'ReactElement<any, string | JSXElementConstructor>', but multiple children were provided.

Reply:
Ok keep it like that then